### DISCUSSION TITLE
What's GL_FOG in Python?

### DISCUSSION THREAD

**Discussion by Leaovo-man2** [community/community#70093] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When I was trying out a game code,An error occurred:"**NameError: name' GL_ FOG 'is not defined**".Excuse me, GL_ What does FOG mean? I can't find any information about him on various websites，This will help me modify the code，. thanks ❤。

**Comment by Leaovo-man2** [comment]
Part of the code looks like this: 
`glEnable (GL-FOG)`
![image](https://github.com/community/community/assets/132978937/650ae9ad-7609-4bca-bd8a-c7526c3e1f7b)

**Reply by Leaovo-man2** [reply]
![image](https://github.com/community/community/assets/132978937/0e07b6ec-1efe-49b3-ad9c-3c999eae0a96)

**Comment by francescoalongi** [comment]
I guess you should provide more info such as which python package you are using, the line triggering this error... Provide these information and maybe people can get to help you.

**Comment by Yjc135** [comment]
python should match the pyglet version, and if the version does not match, the library cannot be referenced. See in detail:[https://blog.51cto.com/u_16175512/12333537](url)
